From f81f9b8bc93a32f5b447a009494ac60355029cfb Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 19 Aug 2016 17:53:32 +0800
Subject: [PATCH 1131/5242] MLK-13096-01 ARM: dts: imx: Update OPP table of
 imx6ull

commit  96c175349ee02cc6e0b21ed2924185e46898e409 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to the latest datasheet(Rev. 0,09/2016), update the
setpoints of i.MX6ULL. we add 25mV margin to cover IR drop
and board tolerance.

LDO enable:
    Freq    VDD_SOC    VDD_ARM
    528MHz  1.175V     1.175V
    396MHz  1.175V     1.025V
    198MHz  1.175V     0.95V

LDO bypass
    Freq    VDD_SOC    VDD_ARM
    528MHz  1.175V     1.175V
    396Mhz  1.175V     1.175V
    198MHz  1.175V     1.175V

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |    6 ++++--
 arch/arm/boot/dts/imx6ull-9x9-evk.dts         |    6 ++++--
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index 3e68c6d..377da6c 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -106,12 +106,14 @@
 	operating-points = <
 		/* kHz	uV */
 		528000	1175000
-		99000	1175000
+		396000	1175000
+		198000	1175000
 	>;
 	fsl,soc-operating-points = <
 		/* KHz	uV */
 		528000	1175000
-		99000	1175000
+		396000	1175000
+		198000	1175000
 	>;
 	arm-supply = <&sw1a_reg>;
 	soc-supply = <&sw1a_reg>;
diff --git a/arch/arm/boot/dts/imx6ull-9x9-evk.dts b/arch/arm/boot/dts/imx6ull-9x9-evk.dts
index f664142..590af08 100644
--- a/arch/arm/boot/dts/imx6ull-9x9-evk.dts
+++ b/arch/arm/boot/dts/imx6ull-9x9-evk.dts
@@ -161,12 +161,14 @@
 	operating-points = <
 		/* kHz	uV */
 		528000	1175000
-		99000	1175000
+		396000	1175000
+		198000	1175000
 	>;
 	fsl,soc-operating-points = <
 		/* KHz	uV */
 		528000	1175000
-		99000	1175000
+		396000	1175000
+		198000	1175000
 	>;
 	arm-supply = <&sw1c_reg>;
 	soc-supply = <&sw1c_reg>;
-- 
1.7.9.5

